x-pack/plugins/security_solution/public/detections/components/rules/rule_actions_overflow/index.tsx

```diff
@@ -96,7 +96,7 @@ const RuleActionsOverflowComponent = ({
             >
               <EuiToolTip
                 position="left"
-                content={getToolTipContent(rule, true, canDuplicateRuleWithActions)}
+                content={getToolTipContent(rule, true, canDuplicateRuleWithActions, true)}
```

Comment:
Could we pass an actual `canUserCRUD` value here instead of the hardcoded `true`?

Reply:
`getToolTipContent()` is used as a utility function here. It can be simplified though full privileges refactoring will be much better.